cves/kernel/CVE-2020-24394.yml

```diff
@@ -157,10 +151,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was originally brought up as a bug on 2020-06-05 by Elliot
+    Mitchell, who was not a Google employee. The vulnerability was found through 
+    manual testing. 
+  automated: false
+  contest: false
+  developer: false
```

Comment:
If it was manually found, then a developer found it, right?

Reply:
Correct, I fixed this.